keps/sig-storage/1790-recover-resize-failure/README.md

```diff
@@ -107,28 +108,68 @@ As part of this proposal, we are mainly proposing three changes:
    - NodeExpansionFailed // state set when expansion has failed in kubelet with a terminal error. Transient errors don't set NodeExpansionFailed.
 3. Update quota code to use `max(pvc.Spec.Resources, pvc.Status.AllocatedResources)` when evaluating usage for PVC.
 
+### Making resizeStatus more general
+
+After some discussion with sig-storage folks and to accommodate changes coming from https://github.com/kubernetes/enhancements/issues/3751 we are proposing that we rename `pvc.Status.ResizeStatus` to `pvc.Status.AllocatedResourceStatus` and make it a map.
+
+So basically new API will look like:
+
+```go
+type ClaimResourceResizeStatus string
```

Comment:
Maybe rename `ClaimResourceResizeStatus` to `ClaimResourceStatus`?

Reply:
fixed.